bookshop/db/init.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * In order to keep basic bookshop sample as simple as possible, we don't add
+ * reuse dependencies. This db/init.js ensures we still have a minimum set of
+ * currencies, if not obtained through @capire/common.
+ */
+
+const cds = require ('@sap/cds')
+const has_common = cds.model.definitions['sap.common.Currencies'].elements.numcode
+
+if (!has_common) module.exports = ()=>
+INSERT.into ('sap.common.Currencies') .columns (
```

Comment:
I think this INSERT will crash on CF w/ HANA the second time it's executed, as there are already records.  Only works on SQlite due to our drop/create approach.

In the future, our 'Initial/test data' story should allow us to deploy data with UPSERTs instead, similar to what HDI does.

Until then we could 
- copy the `Currencies` csv file.  But this could lead to duplicates if bookstore gets deployed
- catch and ignore the DB error here

Reply:
You are right. Re "copy the Currencies csv file" → yes, that was my first approach and causes conflicts
Will go for an additional check.